.mega-linter.yml

```diff
@@ -37,6 +34,8 @@ DISABLE_LINTERS:
   - MARKDOWN_MARKDOWNLINT
   - MARKDOWN_MARKDOWN_TABLE_FORMATTER
   - MARKDOWN_REMARK_LINT
+  - SPELL_PROSELINT
+  - SPELL_CSPELL
```

Comment:
These were previously disabled by L26 but had to now be done this way so we can enable Lychee linter.